src/main/java/in/twizmwaz/cardinal/util/DomUtil.java

```diff
@@ -14,9 +16,23 @@
 
 public class DomUtil {
 
+    private static Document global;
+
+    static {
+        try {
+            byte[] data = IOUtils.toByteArray(Cardinal.getInstance().getResource("global.xml"));
+            File globalFile = new File(Cardinal.getInstance().getConfig().getString("repo") + "/global.xml");
+            org.apache.commons.io.FileUtils.writeByteArrayToFile(globalFile, data);
```

Comment:
No, I believe you can use plugin.getResoure("global.xml") and it will return an input stream. Also why was this not imported?